ajax/libs/react-extras/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "react-extras",
+  "title": "react-extras",
+  "description": "Useful components and utilities for working with React",
+  "author": {
+    "name": "Sindre Sorhus",
+    "url": "sindresorhus.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sindresorhus/react-extras"
+  },
+  "npmName": "react-extras",
+  "npmFileMap": [
+    {
+      "basePath": "source",
```

```suggestion
      "basePath": "dist",
```

`source` dir doesn't exist in package, we want to use `dist`.